function/pkg/docker/runtimes/runtimes.go

```diff
@@ -83,6 +88,37 @@ func ContainerCommands(runtime types.Runtime, debug bool, hotDeploy bool) []stri
 	}
 }
 
+func GetMounts(sourceType workspace.SourceType, workDir string) []mount.Mount {
+	if sourceType == workspace.SourceTypeInline {
+		return []mount.Mount{
+			{
+				Type:   mount.TypeBind,
+				Source: workDir,
+				Target: KubelessTmpPath,
+			},
+			{
+				Type:   mount.TypeVolume,
+				Target: KubelessPath,
+			},
+		}
+	} else {
+		return []mount.Mount{
+			{
+				Type:   mount.TypeBind,
+				Source: workDir,
+				Target: KubelessPath,
+			},
+		}
+	}
+}
+
+func MoveInlineCommand(sourcePath, depsPath string) []string {
+	return []string{
```

Comment:
nit: can we make this a bit more readable? you can put the output of join in variables and then pass them to Sprintf. I think it would be easier to read and understand.